libensemble/manager.py

```diff
@@ -208,6 +230,34 @@ def __init__(
             (1, "stop_val", self.term_test_stop_val),
         ]
 
+        if self.libE_specs.get("manager_runs_additional_worker", False):
```

Comment:
I would think should be an "else" here.

Reply:
I'm not sure what would go in "else", though. We do extra setup for an additional worker, but there's nothing alternative to do otherwise.

Reply:
The lines that are always done above
        else:
            self.W = np.zeros(len(self.wcomms), dtype=Manager.worker_dtype)
            self.W["worker_id"] = np.arange(len(self.wcomms)) + 1